nix/ec2-rds-subnet-group.nix

```diff
@@ -0,0 +1,44 @@
+{ config, lib, uuid, name, ... }:
+
+with lib;
+with import ./lib.nix lib;
+
+{
+  options = {
+    name = mkOption {
+      default = "nixops-${uuid}-${name}";
+      type = types.str;
+      description = ''
+        Name of the RDS DB subnet group.
+      '';
+    };
+
+    description = mkOption {
+      type = types.str;
+      description = ''
+        Description of the RDS DB subnet group.
+      ''; 
+    };
+
+    region = mkOption {
+      type = types.str;
+      description = "Amazon RDS DB subnet group region.";
+    };
+
+    accessKeyId = mkOption {
+      default = "";
+      type = types.str;
+      description = "The AWS Access Key ID.";
+    };
```

Comment:
u can use `imports = [ ./common-ec2-auth-options.nix ];` for these options like we do in the other modules [for example](https://github.com/NixOS/nixops-aws/blob/master/nix/vpc-endpoint.nix)

Reply:
Sure.